README.md

```diff
@@ -46,73 +71,57 @@ This will output similar to the following:
+  * `api-request`: An API Request was needed to be made to the backend server, and this shows the duration that the request took.
+  * `transcribe-json`: JSON Data received from an API request needs to be modified to be displayed by the website. This shows the duration the modification took.
+  * `cache-check`: Local or Remote or Both types of caching are implemented on this endpoint. This shows the duration it took to check with said caches for the relevant data.
 
 Currently `loadStats()` is unable to display how long certain actions took, which are listed below:
+  * The time `ExpressJS` took to handle the request itself.
+  * The time `EJS` templates took to handle creating and displaying the page.
 
-* The time `ExpressJS` took to handle the request itself.
-* The time `PugJS`/`Jade` templates took to handle creating and displaying the page.
+## Microservices
 
----
-
-## Developers
-
-To run and test `package-frontend` locally should be rather simple.
-
-After cloning the repository, run `npm install` within the root of the folder.
-
-Then copy or rename `app.example.yaml` to `app.yaml`. When working with a local app config file you can change settings of the backend API to interact with to use a local version, or change the port the frontend is exposed on.
-
-Within the config file exists the key `GOOGLE_APPLICATION_CREDENTIALS` this key is used to point to a KeyFile containing Google Application Credentails to interact with Remote Caches. If you don't have access to this file set it to `"no-file"` like so:
-
-```yaml
-GOOGLE_APPLICATION_CREDENTIALS: "no-file"
-```
+Some tasks within the frontend are run in their own dedicated microservice. To read more about why some aspects are moved to a microservice and how they behave read [here](./microservices/README.md).
 
-This is a special value that will cause the caching service to automatically ignore any cache requests, and force request through to the backend service. Allowing to fail gracefully.
+### Social Image Cards
 
-### Running In Dev Mode
+Social Image Cards allow the creation of dynamic Package Cards to be displayed, either when linked to directly, or when a link to a packages page on the frontend has been shared to services that display the image websites prompt to.
 
-Using `npm run start:dev` you can run the Package Frontend Server in Development mode. This does two notable things:
-  - Disables Remote Cache Features Natively. Setting `GOOGLE_APPLICATION_CREDENTIALS: "no-file"` is not necessary when run in dev mode.
+To learn more about Social Image Cards, how they behave, and how to contribute take a look at the [Social Image Cards Readme](./microservices/social-cards/README.md).
 
-### Creating a New Theme
+### Download Link
 
-Creating a new theme should be rather simple.
+The Download Link allows easy linking to our most recent Cirrus CI or 'alpha' builds of Pulsar. This lets the worry of finding the relevant link be left to this microservice rather than on contributors.
```

Comment:
```suggestion
The Download Link allows automated linking to our most recent Cirrus CI or 'alpha' builds of Pulsar. This lets the microservice worry about finding the relevant link rather than contributors having to do it by hand.
```
easy -> automated? As its not just easier, we literally dont have to touch it.

Reply:
Fantastic suggestion here